src/utils/__tests__/fixtures/index.js

```diff
@@ -18,3 +19,7 @@ export const mainCategories = makeSelectMainCategories.resultFunc(
 export const subCategories = makeSelectSubCategories.resultFunc(
   makeSelectCategories.resultFunc(state)
 );
+
+// map subcategories to prevent a warning about non-unique keys rendered by input elements 🙄
+export const subcategories = subCategories
```

Comment:
This is confusing; having two consts with the same name and the only difference being the case of one letter. Can't you have the return value of `subCategories` be a mapped collection?

Reply:
I don't get exactly what you mean with a `mapped collection`

subcategories is indeed confusing just like the mapping, I would like to address whole fixture thing, since most of them are outdated, will rename for now